Decred Wallet/Features/More/More Features/Settings/SettingsController.swift

```diff
@@ -64,7 +64,7 @@ class SettingsController: UITableViewController  {
         self.navigationController?.navigationBar.topItem?.rightBarButtonItem = nil
         self.navigationController?.navigationBar.barTintColor = UIColor.appColors.offWhite
         
-        let closeButton = UIBarButtonItem(image: UIImage(named: "ic_close"),
```

Comment:
both icons are useful. the x button should be used when the settings accessed from the splash screen, so you use check if the presentation is modal and display the correct icon